src/admin/python/lsst/qserv/admin/qservCli/launch.py

```diff
@@ -948,18 +955,18 @@ def itest(
         failure, or the returncode of stopping the test database if there was a
         problem doing that, or 0 if there was no problems.
     """
-    ref_db_container_name = itest_ref(qserv_root, itest_file, itest_volume, project, mariadb_image, dry)
+    itest_volumes = make_itest_volumes(project)
+    ref_db_container_name = itest_ref(qserv_root, itest_file, itest_volumes, project, mariadb_image, dry)
     try:
         returncode = integration_test(
             qserv_root,
             itest_container,
-            itest_volume,
+            itest_volumes.exe,
```

Comment:
.exe?

Reply:
yes, itest_volumes is a namedtuple, with 3 members, one of them is `exe`, this is the execution volume (the others are the mariadb data volume, and the mariadb lib volume)